references/video_classification/train.py

```diff
@@ -149,7 +155,12 @@ def main(args):
     print("Loading validation data")
     cache_path = _get_cache_path(valdir)
 
-    transform_test = presets.VideoClassificationPresetEval((128, 171), (112, 112))
+    if not args.weights:
```

Comment:
nit: `pretrained` and `weights` are overlapping and can be confusing. This ideally should be cleaned up in the future

Reply:
That's exactly the plan. `--pretrained` will go away and `--weights` is going to be the right parameter. Right now we support both temporarily so that we can switch between the two completely different APIs. The `--weights` acts as a feature switch here.